test/Autofac.Multitenant.Test/Autofac.Multitenant.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.2;netcoreapp3.0;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;</TargetFrameworks>
```

Comment:
I don't see the point in testing on three different target frameworks? 3.1 should be plenty, right?

Reply:
Agree, was like that before so I kept it. But true, there is nothing framework specific here.